package.json

```diff
@@ -1,23 +1,27 @@
 {
 	"name": "manta-thoth",
 	"description": "Manta-based system for core and crash dump analysis",
-	"version": "1.2.1",
-        "license": "MIT",
+	"version": "1.3.0",
+	"license": "MIT",
 	"author": "Joyent (joyent.com)",
 	"dependencies": {
-		"manta": "^5.0.0",
-		"ctype": ">=0.5.2",
-		"sprintf": "0.1.1",
 		"bunyan": "^1.8.5",
+		"ctype": ">=0.5.2",
 		"jsprim": "^1.3.1",
+		"manta": "^5.0.0",
+		"node-int64": "^0.4.0",
+		"readable-stream": ">=1.1.9",
 		"rethinkdb": "2.0.0",
-		"readable-stream": ">=1.1.9"
+		"smartdc": "^9.0.0",
```

Comment:
I didn't introduce this dependency, but if there's a replacement for sdc-createkey I can use, I could take a look at it?

Reply:
Ah, I see. That dep was already implicitly there. You just correctly made it explicit.  If it works, then I'd keep it for now to avoid more churn.

(The more modern alternative is `triton key add ...` if you like.)

Reply:
Well, if not now, when?

I've replaced this code with triton key instead.